licenses-report.txt

```diff
@@ -0,0 +1,351 @@
+# 3rd-Party Software License Notice
```

Comment:
Is there a maven plugin that generates this on the fly?

Reply:
We should probably add something to CI to make sure it's ran too.

Reply:
@olim7t mentioned that this is what the Java driver is using https://github.com/datastax/java-driver/blob/4.x/pom.xml#L622-L664 

Not sure how we'd configure it differently based on the files

Reply:
The plugin supports inclusion/exclusion filters. We can have multiple executions that operate on specific directories.